src/components/common/form/FormAccessibility.css

```
@@ -0,0 +1,391 @@
+  pointer-events: auto;
+  cursor: pointer;
+}
+
+.react-select-control-area {
+  cursor: pointer;
+  pointer-events: auto;
+}
+
+.react-select-hidden-input {
+  pointer-events: none;
+  tab-index: -1;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)